src/lib/workspace/AnalysisPanel.tsx

```diff
@@ -120,9 +108,11 @@ export function AnalysisPanel(props: Props) {
           },
           {
             display: 'Visualize',
-            // check whether user is at viz's full screen mode
-            route: inFullscreenVisualization
-              ? '/visualizations/pass-through'
+            // check whether user is at viz
+            route: location.pathname
+              .replace(routeBase, '')
+              .includes('/visualizations')
```

Comment:
I think we should keep the replace, but the following can probably be replaced with a `startsWith`:

```suggestion
              .replace(routeBase, '')
              .startsWith('/visualizations')
```

Reply:
@dmfalke Thank you for your suggestion to change and I agree with that to use startsWith() 👍 May I ask your opinion on one more thing ? 😊 I am just debating myself, whether I will stick to the current approach you suggested or another approach to remove the last url element (e.g., using regEx). The former can certainly do what we want now, but the latter can handle more general cases as described follows.

For example, current approach will work for the current case, /viz/pass-through/UUID, but not work for more general case, e.g., /viz/computation/UUID (not currently existing but may probably be added in the future). The second approach (removing the last url element) would be able to handle general cases like `location.pathname.replace(routeBase, '').replace(/\/[^/]*$/, '')`

Certainly we can revisit this by sticking to the current one, though. What do you think?

Reply:
I think this currently approach is actually the behavior we will want. In cases where there are multiple computations offered, clicking on the tab should go to the computation selector.

Reply:
@dmfalke thank you for your info 👍 I have updated with using startsWith() instead of includes()